help/en/html/apps/index.shtml

```diff
@@ -1,23 +1,14 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
-"http://www.w3.org/TR/html4/strict.dtd">
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
 
 <html lang="en">
 <head>
   <meta name="generator" content=
   "HTML Tidy for Mac OS X (vers 31 October 2006 - Apple Inc. build 15.17), see www.w3.org">
 
   <title>JMRI: Applications</title>
-  <!-- Style -->
-  <meta http-equiv="Content-Type" content=
-  "text/html; charset=us-ascii">
-  <link rel="stylesheet" type="text/css" href="/css/default.css" 
-  media="screen">
-  <link rel="stylesheet" type="text/css" href="/css/print.css"
-  media="print">
-  <link rel="icon" href="/images/jmri.ico" type="image/png">
-  <link rel="home" title="Home" href="/">
-  <!-- /Style -->
-</head>
+ <!--#include virtual="../../../../Style.shtml" -->
+
+ </head>
 
 <body>
   <!--#include virtual="../../../../Header.shtml" -->
```

Comment:
You can use "/Header.shtml" from anywhere, "/" will start from the "root" of the server. Less chance of an error

Reply:
Seem to recall a long discussion on this in Issue #7980.  My summary of that 34 item discussion is at the end.  But I will gladly remove the relative addressing in this case.  Thanks.